Flea/Models/Reservation.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+
+namespace Flea.Models
+{
+    public class Reservation
```

Comment:
The class i currently missing a reference to the customer that owns the reservation.